greedy_method/test_knapsack.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import knapsack_problem as kp
+
+
+class TestClass(unittest.TestCase):
+    """
+    Test cases for knapsack_problem
+    """
+
+    def test_sorted(self):
+        """
+        kp.calc_Profit takes the required argument (profit, weight, max_weight)
+        and returns whether the answer matches to the expected ones
+        """
+        gain = kp.calc_Profit([10, 20, 30, 40, 50, 60], [2, 4, 6, 8, 10, 12], 100)
+        # self.assertTrue(result)
+        self.assertEqual(gain, 75)
```

Comment:
```suggestion
        assert gain == 75
```
Pytest does not need all the boilerplate code.

Can we please also have tests that does negative integers, zeros, and floats point numbers so we can test negative user situations?

Reply:
Sure thing! Will add them too!